src/styles/reset.css

```diff
@@ -0,0 +1,5 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
use * selector only for box-sizing. Don't use it for margins and paddings

Reply:
Done. Changed reset file, added a few missed margins